README.md

```diff
@@ -67,3 +68,7 @@ If you want to use the Arduino APIs, you have to do the followings:
 
 - Q: I cannot find the `dankeboy36.vscode-arduino-api` extension in the [VS Code Marketplace](https://marketplace.visualstudio.com/vscode).
 - A: Correct. This solution targets the [Arduino IDE](https://github.com/arduino/arduino-ide). I will publish the VSIX later, when it works in VS Code. By the way, the VSIX is signed with a verified published.
+
+## Acknowledgments
+
+- [@me-no-dev](https://github.com/me-no-dev) for the original [Java implementation](https://github.com/me-no-dev/EspExceptionDecoder) for the Arduino IDE 1.x
```

Comment:
Should this go in the readme of the "arduino-ide-esp-exception-decoder" extension instead?